packages/pharos-site/src/components/statics/color/ColorSwatches.tsx

```diff
@@ -120,10 +153,13 @@ const ColorSwatches: FC<ColorSwatchesProps> = ({
                 {toTitleCase(name.replace(/ base+$/g, '').replace('-', ' '))}
               </div>
               <div>
-                <strong>Hex</strong> - {color.value.toUpperCase()}
+                <strong>Hex</strong> - {Hex}
               </div>
               {!hexOnlyLabel ? (
                 <>
+                  <div>
+                    <strong>HSL</strong> - {color.value}
+                  </div>
```

Comment:
It seemed useful and completionist to include this in the display alongside the hex/RGB/CMYK representations, especially given these are the canonical color definitions.